<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingLSBillPriceDataGUI</class>
 <widget class="QWidget" name="AccountingLSBillPriceDataGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dati computo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="priceGroupBox">
     <property name="title">
      <string>Costi Unitari</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="priceListComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="currentPriceDataSetSpinBoxLabel">
        <property name="text">
         <string>N. Preventivo</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="currentPriceDataSetSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="priceListComboBoxLabel">
        <property name="text">
         <string>Prezzario</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="totalPriceFieldGroupBox">
        <property name="title">
         <string>Campi</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QTableView" name="totalPriceFieldTableView">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
